ui/src/components/SampleQueue/TaskItemContainer.jsx

```diff
@@ -140,10 +140,7 @@ export default function TaskItemContainer({
             />
           )}
         </div>
-        <Collapse
-          id={`collapse-${data.queueID}`}
-          in={Boolean(displayData.collapsed)}
```

Comment:
Removing this `Boolean` type cast now that it's guaranteed to remain a boolean in the state.